_data/glossary/customresourcedefinition.yaml

```diff
@@ -0,0 +1,10 @@
+id: Custom Resource Definition
+name: Custom Resource Definition
+tags:
+- fundamental
+- operation
+full-link: docs/tasks/access-kubernetes-api/migrate-third-party-resource/
+short-description: >
+  A custom resource definition is the representation of a customization of a particular Kubernetes installation.
+long-description: >
+  A resource definition that defines the endpoint in the Kubernetes API that stores a collection of API objects of a certain kind that is specified.
```

Comment:
`that is specified` reads awkwardly, probably can be removed.

Reply:
okay noted!